src/node_api.h

```diff
@@ -512,6 +512,10 @@ NAPI_EXTERN napi_status napi_cancel_async_work(napi_env env,
 // version management
 NAPI_EXTERN napi_status napi_get_version(napi_env env, uint32_t* result);
 
+// Garbage collection
+NAPI_EXTERN int64_t napi_adjust_external_memory(napi_env env,
```

Comment:
I think `napi_status napi_adjust_external_memory(napi_env env, int64_t change_in_bytes, int64_t* result)` would be more consistent with the rest of the API, since almost everything else returns a `napi_status`.